bowling/bowling.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+MAX_PIN = 10
+MAX_FRAME = 10
+BASIC_SIZE_1FRAME = 2
+
+class Frame
+  def initialize(points, previous_frame = nil, pre_previous_frame = nil)
+    @points = points
+    @previous_frame = previous_frame
+    @pre_previous_frame = pre_previous_frame
+  end
+
+  def score
+    @points.sum + score_spare + score_strike
+  end
+
+  def score_spare
+    if !@previous_frame.nil? && @previous_frame.spare?
```

Comment:
[nits]

safe navigation operator （ボッチ演算子） (`&.`) を使うと、よりシンプルに記述することができます。
こちらを使うのはいかがでしょうか？

https://docs.ruby-lang.org/ja/latest/doc/news=2f2_3_0.html

```suggestion
    if @previous_frame&.spare?
```

`@previout_frame`がnilの場合は `@previous_frame&.spare?`が`nil`なので、if文はfalseになります。